# Lacer Fort
#
#	   |--laser
#	rosteps2
#	   |
#	rosteps1
#
#   2048 step = one circle = 360'
#

#laser switch: 0 = off, 1 = on
int16 hit

#stepper motor 1:  stepper motor turn right(+) or left(-)
int16 rotsteps1

#stepper motor 2:  stepper motor turn up(+) or down(-)
int16 rotsteps2

